extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -32,6 +32,7 @@ import {
 } from '../../../../src/service/url-replacements-impl';
 import * as sinon from 'sinon';
 
+/* global require: false */
```

Comment:
Is this for linting? Would this disable this for the entire file or just the following line?